src/server/game/Scripting/ScriptMgr.h

```diff
@@ -1457,7 +1470,12 @@ class ScriptMgr
 
     public: /* SpellSC */ 
  
-        void OnCalcMaxDuration(Aura const* aura, int32& maxDuration); 
+        void OnCalcMaxDuration(Aura const* aura, int32& maxDuration);
```

Comment:
single space was removed here at the end in case someone else is wondering whats happening here